### PR TITLE
Adds emotes.

### PR DESCRIPTION
**Pull request by H0lySquirr3l**
IGNORE THIS

### CODE REVIEW

**Comment by H0lySquirr3l** [comment]
Changed laugh and chuckle to have "v" infront to keep them from conflicting.

File: code/modules/mob/living/carbon/human/emote_vr.dm [hunk 1]
```diff
@@ -130,6 +130,30 @@
...
+			playsound(src, 'sound/voice/Generic-Snort.ogg', 50, 0, preference = /datum/client_preference/emote_noises)
+		if("vlaugh")
+			message = "lets out a laugh."
+			m_type = 2
+			playsound(src, 'sound/voice/Generic-laugh.ogg', 50, 0, preference = /datum/client_preference/emote_noises)
+		if("vchuckle")
+			message = "gives a small chuckle."
+			m_type = 2
+			playsound(src, 'sound/voice/Generic-Chuckle.ogg', 50, 0, preference = /datum/client_preference/emote_noises)
+		if("Ccackles")
```
Comment (KasparoVy) [inline code/modules/mob/living/carbon/human/emote_vr.dm:145]: Ccackles? Is the extra C for extra sekrit?

Comment (H0lySquirr3l) [reply code/modules/mob/living/carbon/human/emote_vr.dm:145]: >-< so much for screets. The C stand for a certian job. Its slightly a longer sounding emote that could be horrible if people found it out and abused it.

Comment (KasparoVy) [reply code/modules/mob/living/carbon/human/emote_vr.dm:145]: Lol, no worries @H0lySquirr3l. In that case I'll delete my comments - if you do the same it'll re-hide this.

**Comment by KasparoVy** [comment]
Damn.

**Comment by H0lySquirr3l** [comment]
Adds
*snort
*moo
*meow
*croak

**Comment by KasparoVy** [comment]
Legendary web editor!

**Pull request closed without merging**